src/Compilers/CSharp/Portable/Binder/Binder_Await.cs

```diff
@@ -431,5 +432,102 @@ private static bool HasOptionalOrVariableParameters(MethodSymbol method)
 
             return false;
         }
+
+        /// <summary>
+        /// Returns the appropriate AsyncMethodBuilder based on whether the
+        /// return type is void or a non-generic or generic task-like type. The
+        /// caller is responsible for ensuring this is used for async methods only.
+        /// </summary>
+        internal AsyncMethodBuilder GetAsyncMethodBuilder(CSharpSyntaxNode syntax, TypeSymbol returnType)
```

Comment:
Changed to no longer use `Binder.LookupMembersInType`. Now lookup simply uses `NamedTypeSymbol.GetMembers(name)`.